tools/docker/mysql/init.d/scaleph-flink-mysql.sql

```diff
@@ -146,4 +146,22 @@ CREATE TABLE flink_job_log
     update_time       DATETIME     NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
     PRIMARY KEY (id),
     UNIQUE KEY uniq_job (flink_job_code, flink_job_version, job_id)
-) ENGINE = INNODB COMMENT = 'flink job log';
\ No newline at end of file
+) ENGINE = INNODB COMMENT = 'flink job log';
+
+DROP TABLE IF EXISTS flink_catalog_configuration;
+CREATE TABLE flink_catalog_configuration
+(
+    id                BIGINT       NOT NULL AUTO_INCREMENT,
+    flink_catalog_type_id BIGINT       NOT NULL,
+    flink_catalog_type_name VARCHAR(64)       NOT NULL,
+    catalog_name VARCHAR(64)       NOT NULL,
+    default_database VARCHAR(64)       NOT NULL,
+    hive_conf_dir VARCHAR(64) default NULL,
+    base_url VARCHAR(64) default NULL,
+    username VARCHAR(64) default NULL,
```

Comment:
default null is useless for mysql itself default behavior, we don't need to add it for clean code

Reply:
sure, I will fix all problem and re-submit a pr later